op-mode-definitions/show-bridge.xml.in

```diff
@@ -42,6 +42,18 @@
             </properties>
             <command>${vyos_op_scripts_dir}/bridge.py show_fdb --interface=$3</command>
           </leafNode>
+          <leafNode name="details">
+            <properties>
+              <help>Display bridge interface details</help>
+            </properties>
+            <command>vtysh -c "show int $3"</command>
+          </leafNode>
+          <leafNode name="nexthop-group">
+            <properties>
+              <help>Display bridge interface nexthop-group</help>
+            </properties>
+            <command>vtysh -c "show int $3 nexthop-group"</command>
```

Comment:
Could you rewrite it with new ` vyos.opmode format`?

Reply:
done